pulsar-functions/secrets/src/main/java/org/apache/pulsar/functions/secretsproviderconfigurator/DefaultSecretsProviderConfigurator.java

```diff
@@ -39,7 +39,9 @@ public String getSecretsProviderClassName(Function.FunctionDetails functionDetai
             case JAVA:
                 return ClearTextSecretsProvider.class.getName();
             case PYTHON:
-                return "secretsprovider.ClearTextSecretsProvider";
+                return "secretsprovider.ClearTextSecretsProvider python";
+            case GO:
```

Comment:
remove this branch here. we should let it throw runtime exception since go doesn't support it.